docs/FAQ/how-to-deploy-bold-bi-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,220 @@
+# How to deploy Bold BI in Elastic Kubernetes Services (EKS) using Application Load Balancer (ALB)
+
+The section explains how to deploy [Bold BI](https://www.boldbi.com/) in an EKS cluster using an [Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
+
+1. Create an Amazon EKS cluster and [node group](https://docs.aws.amazon.com/eks/latest/userguide/eks-compute.html) to deploy Bold BI.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html 
+
+2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following the link below.
+
+   ![AWS EFS](../images/aws-efs.png)
+
+3. Connect to your Amazon EKS cluster.
+   https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-connection/
+  
+4. Deploy the EFS CSI Driver to your cluster.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html 
+
+	```console
+	kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.4"
+	```
+	
+	> <b>Note:</b> Make sure the following network configurations to avoid volume mounting issues.<br/><br/>
+		1. EFS file system in the same region as your EKS cluster.<br/>
+		2. Your EKS cluster and EFS file system must be in the same Virtual Private Cloud (VPC).<br/>
+		3. The security group for your EFS file system, which will allow incoming traffic from your EKS cluster's nodes.<br/>
+		4. Ensure the EFS CSI Driver installation in cluster.<br/>
+		
+5. To install AWS load balancer controller please refer to the below documentation.
```

Comment:
5. To install the AWS load balancer controller, please refer to the following documentation.

Reply:
concern addressed